### ISSUE TITLE
Package.swift parsing errors with Xcode 10.2

### ISSUE THREAD

**Issue by Dschee** [yonaskolb/Mint#129]
I've just upgraded my tool [BartyCrouch]() to Swift 5, including updating the manifest. Now when I try to install it via Mint:

```bash
mint install Flinesoft/BartyCrouch
```

It fails with the following error output:

```bash
🌱  Finding latest version of BartyCrouch
🌱  Cloning BartyCrouch 4.0.1
🌱  Resolving package
🌱  Failed to read Package.swift file:
Couldn't decode package dump:
keyNotFound(CodingKeys(stringValue: "product_type", intValue: nil), Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "products", intValue: nil), _JSONKey(stringValue: "Index 0", intValue: 0)], debugDescription: "No value associated with key CodingKeys(stringValue: \"product_type\", intValue: nil) (\"product_type\").", underlyingError: nil))
```

Running `swift build -c -release` locally doesn't give me any errors.

**Comment by stefanomondino** [comment]
Same issue here with our [Murray](https://github.com/synesthesia-it/Murray).

**Comment by yonaskolb** [comment]
Fixed in 0.12.0
